### ISSUE TITLE
The paras of BertForQuestionAnswering_withIfTriggerEmbedding used in train is right?

### ISSUE THREAD

**Issue by Muzixin** [xinyadu/eeqa#13]
the paras of the BertForQuestionAnswering_withIfTriggerEmbedding‘s forward is 
`def forward(self, input_ids, if_trigger_ids, token_type_ids=None, attention_mask=None, start_positions=None, end_positions=None):`
but the code of train in run_args_qa.py at line 607-610 is
`                    if not args.add_if_trigger_embedding:
                        loss = model(input_ids, segment_ids, input_mask, start_positions, end_positions)
                    else:
                        loss = model(input_ids, segment_ids, if_trigger_ids, input_mask, start_positions, end_positions)`

does the correct？
